services/auto-scan/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:20.04
+
+# Copy local code to the container image.
+ENV HOME /autoscan
+WORKDIR $HOME
+COPY . ./
+
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends apt-utils && \
+apt-get install -y --no-install-recommends python3 python3-pip && \
+apt-get install -y --no-install-recommends python3-setuptools python3-wheel
+
+# Install dependencies.
+RUN python3 -m pip install -r requirements.txt
+
+ENTRYPOINT ["python3", "autoscan.py"]
```

Comment:
If we don't create/specify a user this will end up running as root. Unless someone does something like [userns-remap](https://docs.docker.com/engine/security/userns-remap/) root in the container is going to be root on the host. There is a good example of creating a user [here](https://medium.com/@mccode/understanding-how-uid-and-gid-work-in-docker-containers-c37a01d01cf).